rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/HealthSpecTest.java

```diff
@@ -49,14 +49,15 @@ public void should_document_get_health() throws Exception {
                 .andDo(this.documentationHandler.document(
                         responseFields(
                                 fieldWithPath("status").description("The overall status of the health."),
-                                fieldWithPath("details.diskSpace.status").description("The status of the health of the diskspace."),
-                                fieldWithPath("details.diskSpace.details.total").description("The total volume of the diskspace."),
-                                fieldWithPath("details.diskSpace.details.free").description("The free space of the diskspace."),
-                                fieldWithPath("details.diskSpace.details.threshold").description("The threshold of the diskspace."),
-                                fieldWithPath("details.SW360Rest.status").description("The status of the health of the specific health indicator 'SW360 Rest'."),
-                                fieldWithPath("details.SW360Rest.details.Rest State").description("The details of the rest state in regards to the thrift clients and database."),
-                                fieldWithPath("details.SW360Rest.details.Rest State.isDbReachable").description("The details of the rest state in regards to the thrift clients reachablity represented by a boolean value."),
-                                fieldWithPath("details.SW360Rest.details.Rest State.isThriftReachable").description("The details of the rest state in regards to the databases reachablity represented by a boolean value.")
+                                fieldWithPath("components.diskSpace.status").description("The status of the health of the diskspace."),
+                                fieldWithPath("components.diskSpace.details.total").description("The total volume of the diskspace."),
+                                fieldWithPath("components.diskSpace.details.free").description("The free space of the diskspace."),
+                                fieldWithPath("components.diskSpace.details.threshold").description("The threshold of the diskspace."),
+                                fieldWithPath("components.diskSpace.details.exists").description("The existance of diskspace."),
+                                fieldWithPath("components.ping.status").description("The status of the health of the specific health indicator 'SW360 Rest'.")
+//                                fieldWithPath("details.SW360Rest.details.Rest State").description("The details of the rest state in regards to the thrift clients and database."),
+//                                fieldWithPath("details.SW360Rest.details.Rest State.isDbReachable").description("The details of the rest state in regards to the thrift clients reachablity represented by a boolean value."),
+//                                fieldWithPath("details.SW360Rest.details.Rest State.isThriftReachable").description("The details of the rest state in regards to the databases reachablity represented by a boolean value.")
```

Comment:
Are these fields covered somewhere else?
if yes, can we remove commented code?
if no, can we restore it.

Mismatch in Health Api response

Response Before
```
{"status":"UP","details":{"SW360Rest":{"status":"UP","details":{"Rest State":{"isDbReachable":true,"isThriftReachable":true}}},"diskSpace":{"status":"UP","details":{"total":******,"free":*****,"threshold":*****}}}}
```

Response After
```
{"status":"UP","components":{"SW360Rest":{"status":"UP","details":{"Rest State":{"isDbReachable":true,"isThriftReachable":true}}},"diskSpace":{"status":"UP","details":{"total":*********,"free":**********,"threshold":*******,"exists":true}},"ping":{"status":"UP"}}}
```

Reply:
removed the commented lines.